tendrl/gluster_integration/objects/definition/gluster.py

```diff
@@ -94,6 +94,15 @@
         integration_id:
           help: "Tendrl managed/generated cluster id for the sds being managed by Tendrl"
           type: String
+        integration_name:
```

Comment:
@r0h4n Can u please Tendrl:spec-28-logging-coverage branch? I don't have the permissions I guess